### PR TITLE
[66] [Integrate] As a user, I can pull to refresh surveys

### PR DESCRIPTION
**Pull request by manh-t**
https://github.com/manh-t/flutter-survey/issues/66

## What happened 👀

As a user, I can pull to refresh surveys

## Proof Of Work 📹

https://user-images.githubusercontent.com/60863885/126864469-ac05e2e4-1203-4b27-aaa6-4fe33557c9b6.mp4

### CODE REVIEW

File: lib/pages/dashboard/dashboard_model.dart [hunk 1]
```diff
@@ -93,6 +96,12 @@ class DashboardModel extends StateNotifier<DashboardState> {
     });
   }
 
+  Future<void> refreshSurveys() async {
+    _page = 1;
+    await init();
+    _read(surveyIndexStateProvider).state = 0;
+  }
+
   bool _checkReachedMaxPage() {
```
Comment (markgravity) [inline lib/pages/dashboard/dashboard_model.dart:105]: We can use a getter here like below. What do you think?
```dart
bool get _isMaxPageReached => _page >= _totalPage;
```

**Review by markgravity — APPROVED** [review]
LGTM with small suggestion 👍

File: lib/pages/dashboard/dashboard_model.dart (hunk 1 above)
Comment (manh-t) [reply lib/pages/dashboard/dashboard_model.dart:105]: @markgravity sure 😄 will update 🙏

Comment (manh-t) [reply lib/pages/dashboard/dashboard_model.dart]: @markgravity updated. Thanks 😄

**Review by sleepylee — APPROVED** [review]

**Pull request merged**